monkeys/helm/infinitemonkeys/templates/podmonitor.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: "{{ .Values.service.serverName }}-podmonitor"
+  namespace: {{ .Values.namespace }}
+spec:
```

Comment:
please add the standard helm labels
  labels:
    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
    release: "{{ .Release.Name }}"